tests/test_definitions/column_distributional_expectations/expect_column_kl_divergence_to_be_less_than.json

```diff
@@ -675,7 +675,7 @@
           "catch_exceptions": true
         },
         "out": {
-          "traceback_substring": "ValueError"
```

Comment:
Nope we previously raised a ValueError (we now raise a pydantic.ValidationError).

Regardless, this specific test wasn't great because instead of looking for helpful traceback information around proper arg values, it looked for the word "ValueError" - I think the change made here is for the better